.ci/docker/build.sh

```diff
@@ -379,6 +379,7 @@ case "$image" in
     GCC_VERSION=11
     CONDA_CMAKE=yes
     HALIDE=yes
+    TRITON=yes
```

Comment:
I'm not really sure how `test_halide.py` was passing before, since it does seem to need triton: https://github.com/pytorch/pytorch/blob/f0ba309d78c8ab4930931afccb8afda63cce5ea1/test/inductor/test_halide.py#L224

Reply:
We are only running HalideCPU tests.